### DISCUSSION TITLE
Tabela UserRole

### DISCUSSION THREAD

**Discussion by mdavilag** [balta-io/forum#496] [Tira Dúvidas] [answered]
Bom dia, tudo bem?
Queria tirar uma dúvida. Nessa tabela UserRole que é de relação Many to Many entre a tabela User e a tabela Role, o UserId e o RoleId não deveriam ser Foreign Keys?

Seria na aula: https://balta.io/player/assistir/63c6ef55-44ee-445d-9b53-62e06fc969b8/b6490f05-2417-4a62-b015-17c4d8231bbd

Acesso à dados com .NET, C#, Dapper e SQL Server -> Mão na massa -> Criando as tabelas -> 01:44 de vídeo.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mdavilag deveriam sim! Criando assim uma CONSTRAINT para elas!

**Reply by mdavilag** [reply]
Show, obrigado balta!
